### PR TITLE
app-containers/catatonit: add live & fix symlink

### PR DESCRIPTION
**Pull request by rahilarious**

### CODE REVIEW

File: app-containers/catatonit/metadata.xml [hunk 1]
```diff
@@ -4,7 +4,17 @@
 	<maintainer type="person">
 		<email>zmedico@gentoo.org</email>
 	</maintainer>
+	<maintainer type="project" proxied="proxy">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<maintainer type="person" proxied="yes">
```
Comment (thesamesam) [inline app-containers/catatonit/metadata.xml:11]: please always put proxy-maint last

File: app-containers/catatonit/catatonit-0.1.7-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+src_prepare() {
+	default
+	sed -i -e 's|^AM_INIT_AUTOMAKE$||;' configure.ac || die
+	eautoreconf
+}
+
+src_install() {
+	default
+	dodir /usr/libexec/podman
+	dosym ../../bin/"${PN}" /usr/libexec/podman/"${PN}"
```
Comment (thesamesam) [inline app-containers/catatonit/catatonit-0.1.7-r1.ebuild:30]: Can you use `dosym -r` instead for a nice relative symlink?

File: app-containers/catatonit/catatonit-0.1.7-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+inherit autotools
+
+DESCRIPTION="A container init that is so simple it's effectively brain-dead"
+HOMEPAGE="https://github.com/openSUSE/catatonit"
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/openSUSE/catatonit.git"
+else
+	SRC_URI="https://github.com/openSUSE/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="amd64 arm64 ~ppc64 ~riscv"
```
Comment (thesamesam) [inline app-containers/catatonit/catatonit-0.1.7-r1.ebuild:15]: `pkghceck scan --commits`. Don't commit as stable.

**Review by thesamesam — CHANGES_REQUESTED** [review]

File: app-containers/catatonit/catatonit-0.1.7-r1.ebuild (hunk 2 above)
Comment (rahilarious) [reply app-containers/catatonit/catatonit-0.1.7-r1.ebuild]: neat feature

**Comment by zmedico** [comment]
Thanks!

**Pull request closed without merging**